analysis/import/CSVImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/util/ParserDialogHelper.kt

```diff
@@ -0,0 +1,30 @@
+package de.maibornwolff.codecharta.importer.util
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import java.io.File
+import java.nio.file.Paths
+
+class ParserDialogHelper {
+
+    companion object {
+        fun getInputFiles(isSourceMonitor: Boolean): MutableList<String> {
+            val inputFileNames = mutableListOf(KInquirer.promptInput(
+                    message = if (isSourceMonitor) { "What is the sourcemonitor CSV file that has to be parsed?" } else { "Please specify the name of the first sourcemonitor CSV file to be parsed" },
```

Comment:
The if condition does not makes sence, in both cases "sourcemonitor" is mentioned.

Reply:
I merged the input from the CSVImporter and CSVSourceMonitorImporter here, those were the dialogs as they were before. But I agree with you, I will adjust those questions.